Combobox/ComboPopup.html

```diff
@@ -1,8 +1,9 @@
 <template role="presentation"
 	requires="deliteful/LinearLayout, deliteful/Button">
 	<d-linear-layout style="height: 100%">
-        <div class="d-combo-popup-header">{{header}}</div>
+        <div class="d-combo-popup-header" id="{{_tag}}-{{widgetId}}-header">{{header}}</div>
 		<input d-hidden="{{this.combobox._inputReadOnly}}"
+			aria-labelledby="{{_tag}}-{{widgetId}}-header"
```

Comment:
PS: Oh, I just realized that it isn't checking the `<label>` for the `<input>` in the original Combobox but rather just getting the value, to be used later.